ui/App.tsx

```diff
@@ -87,8 +107,8 @@ const navItems: NavItem[] = [
     icon: IconType.PoliciesIcon,
   },
   {
-    label: "Flux Runtime",
-    link: { value: V2Routes.FluxRuntime },
+    label: "Runtime",
+    link: { value: V2Routes.Runtime },
     icon: IconType.FluxIcon,
```

Comment:
❔ The icon does not change, depending on if the flag is enabled or not, correct?

Reply:
i think it could be the same as per now